server/v2/models/dbQuery.js

```diff
@@ -15,6 +15,7 @@ if (NODE_ENV === 'test') {
 const pool = new Pool({
   connectionString,
 });
+console.log('connectionString1', connectionString);
```

Comment:
@nignanthomas do not push `development tools` with `production code` so always make sure to remove all console.log functions

Reply:
Yes, sorry for that.
I forgot to remove it after debugging my code.